asa/ja-jp/Content/Topics/Adv_Server_Access/docs/ad-user-manage.htm

```diff
@@ -250,11 +250,11 @@ <h3>Active DirectoryパスワードなしID</h3>
                                                         </div>
                                                         <div>
                                                             <h2><a name="map-Okta-user-profile-attributes-to-ASA-user-profile-attributes">Oktaプロファイル属性を</a>アドバンストサーバーアクセス<span class="mc-variable okta-feature-names.ScaleFT variable">のプロファイル属性にマッピングする</span></h2>
-                                                            <p>次に、Oktaプロファイル属性を<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>の属性にマッピングする必要があります。管理者コンソールでプロファイルエディターを開き、<span class="uicontrol">Okta<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>のユーザー</span>プロファイルを参照してから［<span class="uicontrol">Mappings（マッピング）</span>］をクリックします。手順1で作成したOktaユーザー属性から、手順3で作成した<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>属性へのマッピングを作成します。「<a href="https://help.okta.com/okta_help.htm?type=oie&amp;id=csh-usgp-map-attributes" target="_blank">プロファイルエディターでOkta属性をアプリ属性にマッピングする</a>」をご覧ください。</p>
+                                                            <p>次に、Oktaプロファイル属性を<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>の属性にマッピングする必要があります。管理者コンソールでプロファイルエディターを開き、<span class="uicontrol">Okta<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>のユーザー</span>プロファイルを参照してから［<span class="uicontrol">Mappings（マッピング）</span>］をクリックします。手順1で作成したOktaユーザー属性から、手順3で作成した<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>属性へのマッピングを作成します。「<a href="../../../../../../okta_help.htm?locale=ja-JP&amp;type=oie&amp;id=csh-usgp-map-attributes" target="_blank">プロファイルエディターでOkta属性をアプリ属性にマッピングする</a>」をご覧ください。</p>
                                                         </div>
                                                         <div>
                                                             <h2><a name="assign-users-or-groups-to-ASA"></a>ユーザーまたはグループを<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span><br />に割り当てる</h2>
-                                                            <p>前の手順を終えると、属性の構成が完了し、使用を開始できます。<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>に割り当てられたユーザーには、これらの属性が割り当てられます。「<a href="https://help.okta.com/okta_help.htm?type=oie&amp;id=csh-assign-apps" target="_blank">アプリケーションのユーザーへの割り当て</a>」をご覧ください。</p>
+                                                            <p>前の手順を終えると、属性の構成が完了し、使用を開始できます。<span class="mc-variable okta-feature-names.ScaleFT variable">アドバンストサーバーアクセス</span>に割り当てられたユーザーには、これらの属性が割り当てられます。「<a href="../../../../../../okta_help.htm?locale=ja-JP&amp;type=oie&amp;id=csh-assign-apps" target="_blank">アプリケーションのユーザーへの割り当て</a>」をご覧ください。</p>

```

Comment:
This changes should be automated, https://oktainc.atlassian.net/browse/OKTA-538298 should take care of this